pyproject.toml

```diff
@@ -9,12 +9,12 @@ repository  = "https://github.com/nat-n/poethepoet"
 homepage    = "https://github.com/nat-n/poethepoet"
 
 [tool.poetry.dependencies]
-python = ">=3.7"
+python = "^3.7"
```

Comment:
We want to keep compatibility with python >=4 to avoid causing problems for dependent packages. See #43